public/index.html

```diff
@@ -90,6 +90,8 @@ <h1>
 			 * @return {{id: string, token: string}}
 			 */
 			function parseWebhook(url) {
+				if (url.includes('ptb.')) url = url.replace('ptb.', '');
+				if (url.includes('canary.')) url = url.replace('canary.', '');
```

Comment:
You don't need this!
```suggestion
```

Reply:
Instead use a regex capture group, and strip the inital section of the URL!

Reply:
That was my plan - see https://github.com/alii/linear-discord-serverless/pull/12#issuecomment-1103328861